packages/simcore-sdk/src/simcore_sdk/node_ports_common/filemanager.py

```diff
@@ -54,7 +57,7 @@ async def _get_upload_link(
     session: ClientSession,
 ) -> URL:
     presigned_link: AnyUrl = await storage_client.get_upload_file_presigned_link(
-        session, file_id, store_id, user_id
+        session, file_id, store_id, user_id, as_presigned_link=True

```

Comment:
hmm why is there this argument here? the function already implies it is a presigned link.

Reply:
Because it can return an s3 link as well. Renaming function